src/python/pants/backend/codegen/protobuf/python/python_protobuf_subsystem.py

```diff
@@ -72,9 +72,10 @@ class PythonProtobufMypyPlugin(PythonToolRequirementsBase):
         "pants.backend.codegen.protobuf.python",
         "mypy_protobuf_lockfile.txt",
     )
-    default_lockfile_url = git_url(
+    default_lockfile_path = (
```

Comment:
I guess that the `default_lockfile_path` is intentionally undocumented / not part of the interface?

Reply:
Yeah, it's not used by any of our production code, only to facilitate this script. I don't want to require that plugins implement it because they might not need it.